src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts

```diff
@@ -42,8 +42,10 @@ import { GroupTypeUI } from './../../models/group-types.model';
 // ngrx stuff
 import { Store } from '@ngrx/store';
 import { AppState } from './../../states/app.state';
+import * as CustomQueryActions from './../../actions/custom-query.actions';
 import * as FilterActions from './../../actions/filter.actions';
 import * as SpaceActions from './../../actions/space.actions';
+import { CustomQueryModel } from 'src/app/models/custom-query.model';
```

Comment:
@sanbornsen remove this. I think this got added automatically. This causes the unit test to fail.